lib/screens/filters_screen.dart

```diff
@@ -1,18 +1,96 @@
 import 'package:flutter/material.dart';
 import 'package:meals_app/widgets/main_drawer.dart';
 
-class FiltersScreen extends StatelessWidget {
+class FiltersScreen extends StatefulWidget {
   static const routeName = '/filters';
 
+  @override
+  _FiltersScreenState createState() => _FiltersScreenState();
+}
+
+class _FiltersScreenState extends State<FiltersScreen> {
+  var _glutenFree = false;
+  var _vegetarian = false;
+  var _vegan = false;
+  var _lactoseFree = false;
+
+  Widget _buildSwitchListTile(
+    String title,
+    String description,
+    var currentValue,
+    Function updateValue,
+  ) {
+    return SwitchListTile(
+      title: Text(title),
+      value: currentValue,
+      subtitle: Text(description),
+      onChanged: updateValue,
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         title: Text('Your Filters'),
       ),
       drawer: MainDrawer(),
-      body: Center(
-        child: Text('Filters'),
+      body: Column(
+        children: [
+          Container(
+            padding: const EdgeInsets.all(20),
+            child: Text(
+              'Adjust your meal selection',
+              style: Theme.of(context).textTheme.headline1,
+            ),
+          ),
+          Expanded(
+            child: ListView(
+              children: [
+                _buildSwitchListTile(
+                  'Gluten-free',
+                  'Only include gluten-free meals',
+                  _glutenFree,
+                  (newValue) {
+                    setState(
+                      () => _glutenFree = newValue,
+                    );
+                  },
+                ),
+                _buildSwitchListTile(
+                  'Lactose-free',
+                  'Only include lactose-free meals',
+                  _lactoseFree,
+                  (newValue) {
+                    setState(
+                      () => _lactoseFree = newValue,
+                    );
+                  },
+                ),
+                _buildSwitchListTile(
+                  'Vegetarian',
+                  'Only include vegetarian meals',
+                  _vegetarian,
+                  (newValue) {
+                    setState(
+                      () => _vegetarian = newValue,
+                    );
+                  },
+                ),
+                _buildSwitchListTile(
+                  'Vegan',
+                  'Only include vegan meals',
+                  _vegan,
+                  (newValue) {
+                    setState(
+                      () => _vegan = newValue,
+                    );
+                  },
```

Comment:
please see [this comment](https://github.com/MapleSyyrup/Meals_App/pull/15/files#r546781788)